test/src/test/java/org/springframework/security/test/web/servlet/request/SecurityMockMvcRequestBuildersFormLoginTests.java

```diff
@@ -107,9 +112,27 @@ public void postProcessorsAreMergedDuringMockMvcPerform() throws Exception {
 
 		MvcResult mvcResult = mockMvc.perform(formLogin()).andReturn();
 		assertThat(mvcResult.getRequest().getMethod()).isEqualTo(HttpMethod.POST.name());
+		assertThat(mvcResult.getRequest().getHeader("Accept"))
+				.isEqualTo(MediaType.toString(Arrays.asList(MediaType.APPLICATION_FORM_URLENCODED)));
+		assertThat(mvcResult.getRequest().getParameter("username")).isEqualTo("user");
+		assertThat(mvcResult.getRequest().getParameter("password")).isEqualTo("password");
+		assertThat(mvcResult.getRequest().getRequestURI()).isEqualTo("/login");
+		assertCsrfToken(mvcResult.getRequest());
 		verify(postProcessor).postProcessRequest(any());
 	}
 
+	private void assertCsrfToken(MockHttpServletRequest request) {
```

Comment:
I think this assertion is a bit different than some of our other tests. We really only need to ensure that the HTTP parameter `_csrf` exists and is not empty.